ethicml/algorithms/preprocess/pre_algorithm.py

```diff
@@ -32,27 +32,25 @@ def __init__(self, name: str, seed: int, out_size: Optional[int]):
         self._out_size = out_size
 
     @abstractmethod
-    def fit(self, train: DataTuple) -> Tuple[PreAlgorithm, DataTuple]:
```

Comment:
why was the return type like that?

Reply:
🤔  it should only really be the `PreAlgorithm`. I think I did it with both because the training set was always transformed so it feels like a waste to not return it... but this does make the api ugly...

Reply:
is the solution to introduce a `fit_and_transform()`?